### ISSUE TITLE
Crashes if a directory's name looks like source code

### ISSUE THREAD

**Issue by rassie** [cartant/rxjs-report-usage#1]
It's somewhat typical for TypeScript projects to augment third-party code with a typing placeholder. In my case, such a placeholder was in `src/@types/react-clipboard.js/index.d.ts`. `rxjs-report-usage` thought `react-clipboard.js` was a file and crashed with:

```
Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:564:3)
    at tryReadSync (fs.js:349:20)
    at readFileSync (fs.js:386:19)
    at [...]/node_modules/rxjs-report-usage/src/collect-usage.js:23:22
    at Array.forEach (<anonymous>)
    at collectUsage ([...]/node_modules/rxjs-report-usage/src/collect-usage.js:21:17)
    at module.exports ([...]/node_modules/rxjs-report-usage/src/index.js:11:17)
    at Object.<anonymous> ([...]/node_modules/rxjs-report-usage/bin/rxjs-report-usage:6:1)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10) {
  errno: -21,
  syscall: 'read',
  code: 'EISDIR'
}
```

**Comment by cartant** [comment]
Thanks for the bug report. I'll fix this tomorrow.

**Comment by rassie** [comment]
Thanks! I've patched it locally, so my report is already submitted.

**Comment by cartant** [comment]
> ... so my report is already submitted.

Awesome! Thanks.

**Comment by cartant** [comment]
This should be fixed in 1.0.5. Thanks.
